docs/general/stealth-mode.md

```diff
@@ -73,15 +73,15 @@ Some websites and web services still use the old technology to support Java plug
 
 ## Miscellaneous {#miscellaneous}
 
-### Hide Referrer from third-parties {#referrer}
+### Hide Referrer from third parties {#referrer}
 
 Referrer is an HTTP header used in browser-to-server requests. It contains the URL of the request source. When you navigate from one page to another, Referrer saves the URL of the initial page. The server that hosts the destination web page often has software that parses Referrer and extracts various pieces of information from it. Enabling the *Hide Referrer from third-parties* option hides the current website from third-party sites by altering the HTTP header. 
 
 You can also set an arbitrary value for Referrer by entering it into the *Custom Referrer* field. To use default Referrer, leave the field blank.
 
-Note that AdGuard apps route all browser traffic through themselves before sending it to the server. Referrer is getting hidden or substituted after the request leaves the browser but before AdGuard forwards it to the server. This means that if you monitor Referrer inside the browser (for example, with the help of Chrome's Developer Tools), you will see the original Referrer because the request hasn't reached AdGuard yet. You can use software like [Fiddler](https://www.telerik.com/fiddler) to make sure that Referrer gets altered correctly.
+Note that to be able to filter traffic, AdGuard applications 'intercept' browser-to-server requests. Requests to ad, tracking, and phishing servers may be altered before sending them to the server or blocked completely. Same goes for *Hide Referrer from third parties* option: AdGuard intercepts HTTP requests to remove or change the Referrer header if this option is enabled. However, it happens only after these requests leave the browser. This means that if you monitor Referrer inside the browser (for example, with the help of Chrome's Developer Tools), you will see the original Referrer because the request hasn't reached AdGuard yet. You can use software like [Fiddler](https://www.telerik.com/fiddler) to make sure that Referrer gets altered correctly.
```

Comment:
Same goes for the *Hide Referrer from third parties* option:

This means that if you monitor the Referrer inside the browser.

to make sure that the Referrer gets altered correctly.